util/raclgen.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+r"""RACL Generator.
+This utility computes the policy selection vector for a given ip, RACL config, and RACL mapping.
+"""
+
+import argparse
+from io import StringIO
+
+from mako.template import Template
+from raclgen.lib import parse_racl_config, parse_racl_mapping
+from reggen.ip_block import IpBlock
+
+
+def main():
+    parser = argparse.ArgumentParser(prog='raclgen')
+
+    parser.add_argument('--racl-config',
+                        '-r',
+                        required=True,
+                        help='Path to RACL config hjson file.')
+
+    parser.add_argument('--ip',
+                        '-i',
+                        required=True,
+                        help='Path to IP block hjson file.')
+
+    parser.add_argument('--mapping',
+                        '-m',
+                        required=True,
+                        help='Path to RACL mapping hjson file.')
+
+    parser.add_argument(
+        '--if-name',
+        default=None,
+        help=
+        'TLUL path interface name. Required if multiple bus_interfaces exist.')
+
+    args = parser.parse_args()
+
+    try:
+        ip_block = IpBlock.from_path(path=args.ip, param_defaults=[])
+    except ValueError as err:
+        raise SystemExit(
+            f'Failed to parse IP block "{args.ip}". Error: {str(err)}')
+
+    try:
+        parsed_racl_config = parse_racl_config(args.racl_config)
+    except ValueError as err:
+        raise SystemExit(
+            f'Failed to parse RACL config "{args.racl_config}". Error: {str(err)}'
+        )
+
+    parsed_register_mapping, racl_group, policy_names = parse_racl_mapping(
+        parsed_racl_config, args.mapping, args.if_name, ip_block)
+
+    template = """
+<%doc>
```

Comment:
Do we need that as a mako comment? Isn't a python comment enough?

Reply:
But probably to have the same chunk of template in both cases, with the same comment I guess.